src/coreclr/jit/typelist.h

```diff
@@ -3,7 +3,7 @@
 
 #define GCS EA_GCREF
 #define BRS EA_BYREF
-#define PS EA_PTRSIZE
+#define PS TARGET_POINTER_SIZE
```

Comment:
While this makes sense, what's the point of changing this one but leaving the EA_GCREF/EA_BYREF above?

Reply:
GCS and BRS are only used in emit.cpp. Would it be better to `#define` `GCS` and `BRS` in `emit.cpp` ? Or maybe use `EA_GCREF` and `EA_BYREF` directly in the macros below?

Reply:
How about https://github.com/dotnet/runtime/pull/50066 instead? (Easier to code than describe.) It uses emitter `EA_` values in the 2 "emitter" columns, and `TARGET_POINTER_SIZE` in the other columns.

Reply:
Yes, that looks good.